src/renderer/services/notification/index.js

```diff
@@ -0,0 +1,145 @@
+import template from './index.html'
+import './index.css'
+
+const INON_HASH = {
+  primary: 'icon-message',
+  error: 'icon-error',
+  warning: 'icon-warn',
+  info: 'icon-info'
+}
+
+const notification = {
+  name: 'notify',
+  notify ({
+    time = 10000,
+    title = '',
+    message = '',
+    type = 'primary',
+    showConfirm = false
+  }) {
+    let rs
+    let rj
+    let timer = null
+
+    const fragment = document.createElement('div')
+    fragment.innerHTML = template
+      .replace(/\{\{icon\}\}/, INON_HASH[type])
+      .replace(/\{\{title\}\}/, title)
+      .replace(/\{\{message\}\}/, message)
```

Comment:
I'd use DOMPurify to sanitize `title` and `message`. That is most unlikely but just to make sure that no one can load code into renderer.